llvm/lib/Support/RISCVISAInfo.cpp

```diff
@@ -1024,6 +1024,7 @@ static const char *ImpliedExtsZfinx[] = {"zicsr"};
 static const char *ImpliedExtsZhinx[] = {"zhinxmin"};
 static const char *ImpliedExtsZhinxmin[] = {"zfinx"};
 static const char *ImpliedExtsZicntr[] = {"zicsr"};
+static const char *ImpliedExtsZicfiss[] = {"zicsr", "zimop", "zcmop"};
```

Comment:
Zcmop should not be here.

Reply:
Ok I guess the spec does say that, but it seems weird. That would mean that there would be no need for sspush x1 if you can always use c.sspush x1. I'll file an issue on the spec.

Reply:
Here's my issue https://github.com/riscv/riscv-cfi/issues/201#issuecomment-1871383717. From the response I got, I think we should drop Zcmop here and make the CompressPat depend on Zcmop.